server_environment/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "server configuration environment files",
-    "version": "9.0.1.2.0",
+    "version": "10.0.1.2.0",
```

Comment:
This is something we have already discussed, and we got to the agreement of resetting it. Some of the reasons:

* In the migration, you can improve several things from the previous version that don't pair with the previous version.
* There will be for sure in the future version specific patches, and then the pair in the version numbering is a non-sense.
* This eases to see the module in the version progression. If you 10.0.2.0.0, is it because you have made n revisions on v10, or because they were done on previous versions?
* In a couple of Odoo versions more, numbers will be unmanageable (something like 14.0.23.0.2...).